sdks/python/apache_beam/io/avroio.py

```diff
@@ -309,8 +309,8 @@ def _decompress_bytes(data, codec):
 
       # Compressed data includes a 4-byte CRC32 checksum which we verify.
       # We take care to avoid extra copies of data while slicing large objects
-      # by use of a buffer.
-      result = snappy.decompress(buffer(data)[:-4])
```

Comment:
Have you tested this change? When I ran it, it fails with: `TypeError: argument 1 must be string or read-only buffer, not memoryview`.

This is because, a slice of a `buffer` will return the raw data, but in case of `memoryview` a slice will return a `memoryview` object for that sub section.